logout.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FITBIT</title>
+
+    <link rel="stylesheet" href="common.css" />
+    <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+        integrity="sha256-eZrrJcwDc/3uDhsdt61sL2oOBY362qM3lon1gyExkL0=" crossorigin="anonymous" />
+
+    <link rel="stylesheet" href="https://suspicious-sammet-33f1b3.netlify.app/Components/components.css" />
+    <link rel="stylesheet" href="signup.css" />
+</head>
+
+<body>
+    <!--  ******************************** Navbar Start ********************************  -->
+    <nav class="navbar">
+        <div aria-label="menu-icon">
+            <img alt="hamburger-icon" src="images/hamburger.svg" class="hamburger-icon" id="hamburger-icon" />
+        </div>
+        <div class="logo-container" aria-label="fitbit-logo">
```

Comment:
Great that you've used `aria-` property for semantic HTML!

Reply:
Thanks😂